core/src/test/java/io/questdb/test/cutlass/http/line/LineHttpSenderTest.java

```diff
@@ -0,0 +1,386 @@
+                    for (int i = 0; i < totalCount; i++) {
+                        if (i != 0 && i % maxPendingRows == 0) {
+                            serverMain.waitWalTxnApplied("table with space");
+                            serverMain.assertSql("select count() from 'table with space'", "count\n" +
+                                    i + "\n");
+                        }
+                        sender.table("table with space")
+                                .symbol("tag1", "value" + i % 10)
+                                .timestampColumn("tcol4", 10, ChronoUnit.HOURS)
+                                .atNow();
+                    }
+                    serverMain.waitWalTxnApplied("table with space");
+                    serverMain.assertSql("select count() from 'table with space'", "count\n" +
+                            totalCount + "\n");
+                }
+            }
+        });
+    }
+
+    @Test
+    public void testInsertWithIlpHttp() throws Exception {
+        TestUtils.assertMemoryLeak(() -> {
+            try (final TestServerMain serverMain = startWithEnvVariables(
+                    PropertyKey.HTTP_RECEIVE_BUFFER_SIZE.getEnvVarName(), "2048"
+            )) {
+                serverMain.start();
+
+                String tableName = "h2o_feet";
+                int count = 9250;
+
+                sendIlp(tableName, count, serverMain);
+
+                serverMain.waitWalTxnApplied(tableName, 2);
+                serverMain.assertSql("SELECT count() FROM h2o_feet", "count\n" + count + "\n");
+                serverMain.assertSql("SELECT sum(water_level) FROM h2o_feet", "sum\n" + (count * (count - 1) / 2) + "\n");
+            }
+        });
+    }
+
+    @Test
+    public void testInsertWithIlpHttpServerKeepAliveOff() throws Exception {
+        TestUtils.assertMemoryLeak(() -> {
+            try (final TestServerMain serverMain = startWithEnvVariables(
+                    PropertyKey.HTTP_RECEIVE_BUFFER_SIZE.getEnvVarName(), "2048",
+                    PropertyKey.HTTP_SERVER_KEEP_ALIVE.getEnvVarName(), "false"
+            )) {
+                serverMain.start();
+
+                String tableName = "h2o_feet";
+                int count = 9250;
+
+                sendIlp(tableName, count, serverMain);
+
+                serverMain.waitWalTxnApplied(tableName, 2);
+                serverMain.assertSql("SELECT count() FROM h2o_feet", "count\n" + count + "\n");
+                serverMain.assertSql("SELECT sum(water_level) FROM h2o_feet", "sum\n" + (count * (count - 1) / 2) + "\n");
+            }
+        });
+    }
+
+    @Test
+    public void testLineHttpDisabled() throws Exception {
+        Rnd rnd = TestUtils.generateRandom(LOG);
+        TestUtils.assertMemoryLeak(() -> {
+            int fragmentation = 1 + rnd.nextInt(5);
```

Comment:
`fragmentation` looks like a leftover from another test copy as it's unused here